index.html

```diff
@@ -9,7 +9,7 @@
 
 <body>
         <div class="section" id="plan">
-                <h1>Practicing Git</h1>
+                <h1>Hacking Open Source at Columbia</h1>

```

Comment:
This doesn't make sense...it should be Hacking Open Source at Columbia featuring OpenHatch. Maybe @jcf2167 can take a look!